tests/data_test.py

```diff
@@ -462,3 +463,23 @@ def test_tabular_transform() -> None:
     tf2 = ZScoreNormalize()
     with pytest.raises(RuntimeError):
         test.transform_(tf2)  # Not yet fitted.
+
+
+def test_acs_datamodule() -> None:
+    acs_income = ACSDataModule(setting=ACSDataset.Setting.income)
+    acs_income.prepare_data()
+    acs_income.setup()
+    assert acs_income.train_data.feature_groups is not None
+    assert acs_income.train_data.feature_groups[0] == slice(2, 10)
+    assert acs_income.train_data.feature_groups[1] == slice(10, 33)
+    assert acs_income.train_data.x.shape == (13_360, 729)
+    assert acs_income.train_data.s.shape == (13_360,)
+    assert acs_income.train_data.y.shape == (13_360,)
+    assert acs_income.train_data.non_ohe_indexes == [0, 1]
+
+    # Verify that the non-one-hot-encoded features are normalized.
+    assert acs_income.train_data.x[:, 0].mean() == pytest.approx(0.0, abs=1e-5)
```

Comment:
Could also check for unit variance I suppose but I think the test is sufficiently rigorous as it is